packages/wrangler/src/deploy/index.ts

```diff
@@ -361,6 +363,15 @@ export async function deployHandler(
 		experimentalVersions: args.experimentalVersions,
 	});
 
+	if (deploymentId) {
+		writeOutput({
+			type: "deployment",
+			version: 1,
+			worker_id: name,
+			deployment_id: deploymentId,
+		});
+	}
```

Comment:
I think you intended to have something like this for versions upload in versions/index.ts (which matches this deploy/deploy.ts + deploy/index.ts file structure)